tool/bin/devtools_tool.bat

```diff
@@ -0,0 +1,3 @@
+pushd "%~dp0"
+dart run ./devtools_tool.dart %*
```

Comment:
will the bat fail if the dart run fails? If there is a way to make sure that happens it would be a good thing to have :)